q2_feature_classifier/_vsearch.py

```diff
@@ -81,3 +192,57 @@ def classify_consensus_vsearch(query: DNAFASTAFormat,
                  'choosing the top N hits, not the first N hits.'),
     citations=[citations['rognes2016vsearch']]
 )
+
+
+plugin.pipelines.register_function(
+    function=classify_hybrid_vsearch_sklearn,
+    inputs={**inputs, 'classifier': TaxonomicClassifier},
+    parameters={**parameters,
+                'reads_per_batch': Int % Range(0, None),
+                'confidence': Float,
+                'read_orientation': Str % Choices(
+                    ['same', 'reverse-complement']),
```

Comment:
Should there be an `'auto'` choice, too? I recognize that that doesn't quite match up with the parent Action's API, but maybe this could let us move in the right direction.

Reply:
what would 'auto' do? I agree it would be a move in the right direction but right now the `classify_sklearn` method has no such option.

Reply:
👍